message_definitions/v1.0/ras_a.xml

```diff
@@ -220,5 +382,68 @@
+      <field type="float" name="egress_portal_alt" units="m" invalid="NaN">Currently defined egress portal Altitude (MSL). NaN if unknown, not applicable or when egress_portal_id is set and different from UINT32_MAX.</field>
+    </message>
+    <message id="452" name="EXPLORATION_RETURN_POSITION">
+      <wip/>
+      <!-- This message is work-in-progress and it can therefore change. It should NOT be used in stable production environments. -->
+      <description>Provides the return-from-exploration position when an exploration is completed (e.g. volume set by the exploration boundaries does not have new open areas for
+        the vehicle to explore) or canceled (e.g. the operator stops the exploration task and requests the vehicle to leave the defined exploration area).
+        Can either be set by the vehicle's onboard autonomy engine or set by the user MAV_CMD_SET_EXPLORATION_RETURN_POS.
+        A MAV_CMD_NAV_EXPLORATION_RETURN can be used to send the vehicle to the position defined by this message.
+        This message can be requested by sending the MAV_CMD_REQUEST_MESSAGE command.
+      </description>
+      <field type="uint64_t" name="time_usec" units="us" invalid="0">Timestamp (UNIX Epoch time or time since system boot). The receiving end can infer timestamp format (since 1.1.1970 or since system boot) by checking for the magnitude of the number.</field>
+      <field type="int32_t" name="latitude" units="degE7" invalid="INT32_MAX">Latitude (WGS84). INT32_MAX when unknown.</field>
+      <field type="int32_t" name="longitude" units="degE7" invalid="INT32_MAX">Longitude (WGS84). INT32_MAX when unknown.</field>
+      <field type="int32_t" name="altitude" units="mm" invalid="NaN">Altitude (MSL). Positive for up. Note that virtually all GPS modules provide both WGS84 and MSL. INT32_MAX when unknown.</field>
+      <field type="int32_t" name="relative_alt" units="mm" invalid="NaN">Altitude above ground. INT32_MAX when unknown.</field>
+      <field type="int32_t" name="geoid_alt" units="mm" invalid="NaN">Altitude relative to WGS84 geoid. INT32_MAX when unknown.</field>
+      <field type="float" name="x" units="m" invalid="NaN">Local X position of this position in the local coordinate NED frame. NaN when unknown.</field>
+      <field type="float" name="y" units="m" invalid="NaN">Local Y position of this position in the local coordinate NED frame. NaN when unknown.</field>
+      <field type="float" name="z" units="m" invalid="NaN">Local Z position of this position in the local coordinate NED frame. NaN when unknown.</field>
+      <field type="float" name="yaw" units="rad" invalid="NaN">World to surface heading transformation of the return-from-exploration position. Used to indicate the heading with respect to the ground. NaN when unknown.</field>
```

Comment:
NOw you made this just yaw, the description needs to change.

Reply:
Because when I read "World to surface heading transformation" I don't think "yaw". Who would?